frontend/src/container/Header/index.tsx

```diff
@@ -109,9 +110,13 @@ function HeaderContainer(): JSX.Element {
 
 	const { data: licenseData, isFetching, status: licenseStatus } = useLicense();
 
+	const licensesStatus: string =
+		licenseData?.payload?.licenses?.find((e: License) => e.isCurrent)?.status ||
+		'';
+
 	const isLicenseActive =
-		licenseData?.payload?.licenses?.find((e) => e.isCurrent)?.status ===
-		LICENSE_PLAN_STATUS.VALID;
```

Comment:
Can we do something like:

`status.toLocalLowerCase() === LICENSE_PLAN_STATUS.VALID.toLocaleLowerCase()` just to ensure that a case insensitive check is happening?

Reply:
we are doing that in the next line ?

Reply:
That was my point, my apologies, I didn't read it correctly.